extensions/3DTILES_layers/README.md

```diff
@@ -0,0 +1,98 @@
+# 3DTILES_layers
+
+## Overview
+
+This extension to 3D Tiles enables grouping of different types of content into layers. A layer is a user-defined grouping of content - for example, a tileset of a city may have a buildings layer, a roads layer and a vegetation layer. Each layer follows an independent layout for its tiles and there are not dependencies between layers. A layer may also associate with itself some metadata by conforming to a `class` declared in `3DTILES_metadata`.
+
+## Contributors
+
+Cesium
+
+## Schema Changes
+
+### Layer Metadata
+
+Layers are declared inside the top-level `3DTILES_layers` extension dictionary. Each layer is an object with its key being the layer's ID. The layer may declare a `name` and a `description`. 
+
+```javascript
+{
```

Comment:
It might be nice if all examples in this document have the three layers described in the overview: a buildings layer, a roads layer and a vegetation layer (or rename to "trees" layer). I don't think one layer is enough to convey the extension.

Reply:
Added `buildings`, `trees`, `roads` as instances of `cityLayer`.